source/GameInterface/Services/Settlements/Patches/SettlementHitPointsPatch.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Common;
+using Common.Messaging;
+using Common.Util;
+using GameInterface.Extentions;
+using GameInterface.Policies;
+using GameInterface.Services.Settlements.Messages;
+using HarmonyLib;
+using System.Reflection;
+using TaleWorlds.CampaignSystem.Settlements;
+
+namespace GameInterface.Services.Settlements.Patches;
+
+
+/// <summary>
+/// Sync the SettlementHitPoints
+/// </summary>
+[HarmonyPatch(typeof(Settlement))]
+internal class SettlementHitPointsPatch
+{
+
+    [HarmonyPatch(nameof(Settlement.SettlementHitPoints), MethodType.Setter)]
+    [HarmonyPrefix]
+    private static bool SettlementHitPointsPrefix(ref Settlement __instance, ref float value)
+    {
+        if (AllowedThread.IsThisThreadAllowed()) return true;
+        if (CallOriginalPolicy.IsOriginalAllowed()) return true;
+
+        if (ModInformation.IsClient) return false;
```

Comment:
Change this to match up to line 42 of https://github.com/Bannerlord-Coop-Team/BannerlordCoop/blob/development/source/GameInterface/Services/Armies/Patches/ArmyLifetimePatches.cs#L42